packages/react-components/lib/map/three-fiber/robot-three-maker.tsx

```diff
@@ -1,7 +1,7 @@
-import { Circle, Line, Text } from '@react-three/drei';
 import { ThreeEvent } from '@react-three/fiber';
 import React from 'react';
 import { Euler, Vector3 } from 'three';
+import { CircleShape } from '.';
```

Comment:
Typescript should be able to handle this, but we should try not to import in a circular fashion anyway (robot-three-maker->index->robot-three-maker)

Reply:
Done 9e45765c0b014774b72ef8908b5316d612c220cf